x-pack/legacy/plugins/uptime/public/components/functional/monitor_page_link.tsx

```diff
@@ -20,13 +20,14 @@ export const MonitorPageLink: FunctionComponent<DetailPageLinkProps> = ({
   location,
   linkParameters,
 }) => (
+  // encode monitorid param as 64 base string to make it a valid URL, since it can be a url
   <EuiLink>
     <Link
       data-test-subj={`monitor-page-link-${id}`}
       to={
         location === undefined
-          ? `/monitor/${id}${linkParameters}`
-          : `/monitor/${id}/${encodeURI(location)}/${linkParameters}`
+          ? `/monitor/${btoa(id)}${linkParameters}`
```

Comment:
Related to the nitpick about the comment above:

```suggestion
          ? // encode monitorid param as 64 base string to make it a valid URL, since it can be a url
            `/monitor/${btoa(id)}${linkParameters}`
```

I put this comment above the ternary operator and prettier moved it like this.

I'd also be fine with putting the comment above the equality operation.

Reply:
moved the comment closer to implementation.